app/views/application/_logos.html.haml

```diff
@@ -0,0 +1,7 @@
+.row
+  .col.s12
+    - if Rails.application.config.settings.logo.company_logo.present?
+      = image_tag Rails.application.config.settings.logo.company_logo, style: 'width: 20%; display: inline; vertical-align: middle'
+      = image_tag logo_image, style: 'width: 15%; display: inline; vertical-align: middle; margin-left:63.5%'
+    - else
+      = image_tag logo_image, style: 'width: 20%; margin: auto; display: block;', class: 'right-align'
```

Comment:
ik wou m er de vorige keer al bij zetten, maar misschien nog even een `if logo_image.present?` hierbij. Voor als we geen logo boven aan de pagina willen

Reply:
Ik vind m hip

Reply:
Fixed.